src/Application/Apps/Queries/GetAppsQuery.cs

```diff
@@ -16,20 +16,23 @@ public class GetAppsQueryHandler : IRequestHandler<GetAppsQuery, AppsVm>
 
     private readonly IMapper _mapper;
 
-    public GetAppsQueryHandler(IApplicationDbContext context, IMapper mapper)
+    public GetAppsQueryHandler(IApplicationDbContext context,
```

Comment:
In general ar fi ok sa nu comituim fisiere care practic nu s-au modificat. Cum e asta.